lib/membrane_live/event.ex

```diff
@@ -202,6 +186,34 @@ defmodule MembraneLive.Event do
     {:noreply, state}
   end
 
+  @impl true
+  def handle_info(%Message.EndpointCrashed{endpoint_id: "hls_output"}, state) do
+    Membrane.Logger.error("HLS endpoint has crashed!")
```

Comment:
in this case I would also log information about restarting hls_endpoint

Reply:
Log about recreating HLS endpoint is in other callback.